native/SampleApps/SmartSyncExplorer/SmartSyncExplorer/Classes/AppDelegate.m

```diff
@@ -129,7 +130,7 @@ - (void)setUserLoginStatus :(BOOL) loggedIn {
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-    self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+    self.window = [[SFSDKUIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
```

Comment:
Do we need these changes in the template apps too?

Reply:
@bhariharan IMO having it here is a good for us to test rotations and should also serve as an example for developers. I have added notes here.